util/imagetools/inspect.go

```diff
@@ -168,5 +168,6 @@ func (r *Resolver) ImageConfig(ctx context.Context, in string, platform *ocispec
 	if err != nil {
 		return "", nil, err
 	}
-	return imageutil.Config(ctx, in, r.resolver(), r.buffer, nil, platform)
+	_, dgst, data, err := imageutil.Config(ctx, in, r.resolver(), r.buffer, nil, platform, nil)
```

Comment:
This function is dead code now, we should be good to remove it.

Reply:
I noticed that! Wanted to double-check if it was used anywhere else, but I can remove it

Reply:
Addressed in https://github.com/docker/buildx/pull/1950